rust/src/smb/smb.rs

```diff
@@ -1259,7 +1247,6 @@ impl SMBState {
                 if let Ok((_, ref hdr)) = parse_nbss_record_partial(input) {
                     if !hdr.is_smb() {
                         SCLogDebug!("partial NBSS, not SMB and no known msg type {}", hdr.message_type);
-                        self.trunc_ts();
```

Comment:
I'm missing a discussion of this removal. It's not related to stream depth.

Reply:
oh dear. there are two call paths to this. Totally missed it.